example/01_gemm/gemm_dl_int4.cpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#ifndef CK_EXPERIMENTAL_BIT_INT_EXTENSION_INT4
+#error Should compile this file with ck::int4_t support
+#endif
```

Comment:
I wonder whether this is necessary, since this file compilation is already guarded appropriately in CMakeLists.txt